bindings/ergo-lib-c/cbindgen.toml

```diff
@@ -140,7 +141,7 @@ extra_bindings = []
 crates = ["ergo-lib-c"]
 all_features = false
```

Comment:
I've checked all the changes and nothing caught my eye. Although I have not used crate features in C-Swift bindings ever. As a backup plan, maybe `all_features = true` would work.